packages/components/src/Components/FilterHooks/constants.js

```diff
@@ -52,7 +52,11 @@ export function constructGroups(allTags, item = 'item') {
         items: tags.map(({ count, tag: { key: tagKey, value } }) => ({
             className: 'ins-c-tagfilter__option',
             label: <React.Fragment>
-                <div className="ins-c-tagfilter__option-value">{tagKey}{value ? `=${value}` : ''}</div>
+                <Tooltip
+                    content={`${tagKey}${value ? `=${value}` : ''}`}
```

Comment:
just a nitpick: this string could be calculated in a variable and be used at both places

Reply:
Yeah, good point. I was thinking about that as well, but didn't wanted to change this to function body. But it might be better to use constant, just for sanity and future improvements.